lib/SlackWebClient.js

```diff
@@ -0,0 +1,68 @@
+/** A simple Slack WebAPI client. */
+const SLACK_API_URL = 'https://slack.com/api/';
+
+class SlackWebClient {
+
+    constructor(botToken) {
+        this._botToken = botToken;
+    }
+
+    /** Get all conversations the app is a member of. */
```

Comment:
conversations refers to channels? Why the conversation wording? I assume because the API calls it `users.conversations`?

I would simplify here and call it channels everywhere but the API call. This right now is ambiguous between channels and threads for me.

Reply:
Yes, there is also other types of conversations, like DMs and group chats.

I have changed the wording.